Update Java documentation (#28)
undefinedlabs/scope-docs: docs/java-runtime.md

---
id: java-runtime
title: Scope Java Agent runtime instrumentation
sidebar_label: Runtime instrumentation
---

In order to see trace information from your Java service on integration and end-to-end tests,
you need to use the Scope agent to instrument your running service.

This service might run, for example, in a container on CI, or in a QA/staging environment.

## Using `javaagent` VM param 

> Download the Scope Java Agent, changing version if it is necessary.

Reviewer: ```suggestion
Download the Scope Java Agent, replacing `0.1.9` with the latest version.
```

```bash
cd /path/to/scope/agent
curl -s -o scope-agent.jar "https://repo1.maven.org/maven2/com/undefinedlabs/scope/scope-agent/0.1.9/scope-agent-0.1.9.jar"
```

> Create / Modify `scope.yml` on the app root folder to disable testing mode

Reviewer: Testing mode should be disabled by default, right? And it should be automatically enabled if we detect we are running on CI?

Author: Right. I'm removing this part.
